### ISSUE TITLE
The section on the "delete" command in the User Guide can benefit from a "before" image

### ISSUE THREAD

**Issue by J-wang-CSMA** [J-wang-CSMA/pe#10]
# Expected
A pair of images, before and after to showcase what has been deleted and how the UI changes.

# Actual
Only the after image was provided. This can lead to new users not being aware of how the information displayed on the GUI changed according to the command.

![image.png](https://raw.githubusercontent.com/J-wang-CSMA/pe/main/files/86fd8297-ed87-46da-8ae5-496854de7f0f.png)

**Comment by nus-pe-script** [comment]
# Team's Response
No details provided by team.

## The 'Original' Bug

[The team marked this bug as a duplicate of the following bug]

># In the UG, none of the features have before/after images
>
>**Note from the teaching team:** This bug was reported during the _Part II (Evaluating Documents)_ stage of the PE. **You may reject this bug if it is not related to the quality of documentation.**<hr>
>It can be hard to follow some commands like 'edite' 'deletee' and it would be useful if the UG showed what was displayed before and after the command was executed.
>
>
>
>
>
><hr>
><sub>[original: nus-cs2103-AY2324S2/pe-interim#3960] [original labels: severity.Low type.DocumentationBug]</sub>
>
>
>

## Their Response to the 'Original' Bug

[This is the team's response to the above 'original' bug]

>Hi, thanks for pointing this out :3
>
>However, we have already added a note in the UG that the state of the data before each example is that of the original sample data. We believe that adding a before / after image for all our features would actually clutter the UG more, and hence, we made the decision to format our UG in this manner.
>
>![image.png](https://raw.githubusercontent.com/nus-cs2103-AY2324S2/pe/master/files/b374d68f-489e-4599-8d26-e57146d08563.png)
# Items for the Tester to Verify
## :question: Issue duplicate status

Team chose to mark this issue as a duplicate of another issue (as explained in the _**Team's response**_ above)

- [ ] I disagree

**Reason for disagreement:** [replace this with your explanation]

<catcher-end-of-segment><hr>
## :question: Issue response

Team chose [`response.Rejected`]

- [x] I disagree

**Reason for disagreement:** While the group have stated that the "before" data is the original state of the base sample data, the original state of the base sample data is changed after any command the user uses to modify the list. Hence, the audience can only refer to the original state once (when they first open up the application) when they test out the commands, which still demands some kind of before image or at least some markings to clearly indicate what has changed with the command given.



<catcher-end-of-segment><hr>
## :question: Issue severity

Team chose [`severity.VeryLow`]
Originally [`severity.Low`]

- [x] I disagree

**Reason for disagreement:** This is not just a cosmetic or formatting issue.

<catcher-end-of-segment><hr>
